docs/config/README.md

```diff
@@ -3477,14 +3498,14 @@ To enable it, set `disabled` to `false` in your configuration file.
 
 | Option                      | Default                                                                       | Description                                                           |
 | --------------------------- | ----------------------------------------------------------------------------- | --------------------------------------------------------------------- |
-| `format`                    | `"[$symbol$status]($style) "`                                                 | The format of the module                                              |
-| `symbol`                    | `"❌"`                                                                         | The symbol displayed on program error                                 |
-| `success_symbol`            | `""`                                                                          | The symbol displayed on program success                               |
-| `not_executable_symbol`     | `"🚫"`                                                                         | The symbol displayed when file isn't executable                       |
-| `not_found_symbol`          | `"🔍"`                                                                         | The symbol displayed when the command can't be found                  |
-| `sigint_symbol`             | `"🧱"`                                                                         | The symbol displayed on SIGINT (Ctrl + c)                             |
-| `signal_symbol`             | `"⚡"`                                                                         | The symbol displayed on any signal                                    |
-| `style`                     | `"bold red"`                                                                  | The style for the module.                                             |
+| `format`                    | `'[$symbol$status]($style) '`                                                 | The format of the module                                              |
+| `symbol`                    | `'❌'`                                                                         | The symbol displayed on program error                                 |
+| `success_symbol`            | `''`                                                                          | The symbol displayed on program success                               |
+| `not_executable_symbol`     | `'🚫'`                                                                         | The symbol displayed when file isn’t executable                       |
```

Comment:
As far as I can tell, isn't `'` the proper character to use for an apostrophe?

Reply:
This is about the replacement of `'` with `’` as the supposedly correct character, right?

Looking at [Wikipedia Apostrophe](https://en.wikipedia.org/wiki/Apostrophe) and specifically [the section typographic form](https://en.wikipedia.org/wiki/Apostrophe#Typographic_form), it seems that both are useable and correct; where `’` is the explicit apostrophe, and `'` is a generalized character that can be used for an apostrophe amongst other meanings/symbol generalization.

> With the invention of the [typewriter](https://en.wikipedia.org/wiki/Typewriter), a "neutral" or "straight" shape quotation mark, ', was created to represent a number of different glyphs with a single keystroke: the apostrophe, both the opening and the closing single [quotation marks](https://en.wikipedia.org/wiki/Quotation_mark), the single [primes](https://en.wikipedia.org/wiki/Prime_(symbol)), and on some typewriters even the [exclamation point](https://en.wikipedia.org/wiki/Exclamation_point) (by backspacing and overprinting with a period). This is known as the typewriter apostrophe or 'vertical apostrophe'.

It seems I was misled a bit about only one being explicitly, typographically correctness, as [that is the case in German](https://de.wikipedia.org/wiki/Apostroph#Typografisch_korrekt).

I think there would still be value in using an apostrophe over the typewriter apostrophe for differentiation to the code single quote - but it is somewhat questionable to do so only in this document and not everywhere leading to inconsistencies, and when it is not explicitly incorrect to use either. And when one, presumably, sees `'` in en texts more often.

Shall I remove the apostrophe change commit form this PR?

Reply:
Please remove that change. I actually only looked up the Unicode definitions, and "apostrophe" seemed to be preferable over any "right quotation mark".